db/queries/application/queries.py

```diff
@@ -178,6 +183,23 @@ def get_count_by_status(round_ids: Optional[list] = [], fund_ids: Optional[list]
     return results
 
 
+def create_qa_base64file(application_data: dict, with_questions_file):
+    if with_questions_file is not None and with_questions_file:
+        fund_details = get_fund(application_data["fund_id"])
+        q_and_a = extract_questions_and_answers(application_data["forms"], application_data["language"])
+        contents = BytesIO(
+            bytes(generate_text_of_application(q_and_a, fund_details.name, application_data["language"]), "utf-8")
+        ).read()
+        if len(contents) > Config.DOCUMENT_UPLOAD_SIZE_LIMIT:
```

Comment:
How likely is this? What are we doing if we hit this error? Or is it handled by the calling class?

Reply:
Fixed